packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -27,20 +22,37 @@
 						size="small"
 					/>
 				</div>
-				<div v-else :class="issues.length && !hideIssues ? $style.hasIssues : $style.input">
+				<div
+					v-else
+					:class="issues.length && !hideIssues ? $style.hasIssues : $style.input"
+					data-test-id="node-credentials-select"
+				>
 					<n8n-select
 						:value="getSelectedId(credentialTypeDescription.name)"
-						@change="(value) => onCredentialSelected(credentialTypeDescription.name, value)"
+						@change="
+							(value) =>
+								onCredentialSelected(
+									credentialTypeDescription.name,
+									value,
+									showMixedCredentials(credentialTypeDescription),
+								)
+						"
```

Comment:
Nested functions in an inline call gets a bit hard to follow.

Reply:
Agree, but need all these call here in the loop. This component will be reworked in P1 of this project, so keeping it.